code/game/objects/structures/signs.dm

```diff
@@ -574,12 +574,12 @@
 
 /obj/structure/sign/faction/excelsior
 	name = "EXCELSIOR"
-	desc = "EXCELSIOR is union of communist mining communities with no respect to any private property."
+	desc = "EXCELSIOR is a union of communist mining communities with no respect to any private property."
```

Comment:
an union

Reply:
No. Technically it would be a not an.
![image](https://user-images.githubusercontent.com/6728268/69892140-5c582700-12d0-11ea-93b8-979b905be199.png)

Reply:
But it states ``case for many other words starting with a vowel"
``
and it then has one with a u at the start...

Reply:
It's about pronunciation is the thing. you use "an" when the proceeding word starts with the _sound_ of a vowel (A, E, I, O or U)

You use a in this case because when you pronounce the word "union," even though the word is spelled with a u, the sound you make when you say it it starts with a hard 'y', like in the words "yellow" or "you."

English is a dumb language.

Reply:
Also, should say "EXCELSIOR is a union of communist mining communities with no respect **for** any private property," not to.